### PR TITLE
Added custom Hosts to MGT Providers

### PR DESCRIPTION
**Pull request by techienickb**
Updated the Graph Toolkit providers and msal2 provider readme files with customHosts attribute information.

Re-submission as per https://github.com/microsoftgraph/microsoft-graph-docs-contrib/issues/978

### CODE REVIEW

**Comment by jasonjoh** [comment]
@gavinbarron @sebastienlevert Can you please review Nick's resubmission?

**Comment by Lauragra** [comment]
@techienickb , FYI, it will be easier for you to contribute if you follow the Get Started steps to set yourself up to be a contributor. Then you can open your PR in the authoring repo. This repo is a public mirror of the authoring repo. Please ping me internally if you need a link to the Get started steps. Thanks.

**Comment by techienickb** [comment]
@Lauragra I'm not a MS employee, just a contributor to the graph toolkit.  I'm doing what I was requested to do with from #978

**Comment by sebastienlevert** [comment]
@Lauragra @jasonjoh I assume this repo is absolutely OK for external members, right? @techienickb is an awesome contributor for MGT but is not a FTE. We should accept this PR.

**Review by sebastienlevert — APPROVED** [review]

File: concepts/toolkit/providers/msal2.md [hunk 1]
```diff
@@ -130,6 +133,34 @@ Alternatively:
...
+```html
+<mgt-msal2-provider
+      client-id="2dfea037-xxx-c05708a1b241"
+      redirect-uri="http://localhost:3000"
+      custom-hosts="mydomain.com"
+      scopes="user.read,user.read.all">
+</mgt-msal2-provider>
+```
+
+
```
Comment (gavinbarron) [inline concepts/toolkit/providers/msal2.md:163]: This is great, the think I think it needs to complete this documentation is an example of then using this provider when calling to your custom Entra secured API.

**Review by gavinbarron — CHANGES_REQUESTED** [review]

File: concepts/toolkit/providers/msal2.md (hunk 1 above)
Comment (techienickb) [reply concepts/toolkit/providers/msal2.md:179]: @gavinbarron added an example of using it

**Comment by jasonjoh** [comment]
> @Lauragra @jasonjoh I assume this repo is absolutely OK for external members, right?

Yes, absolutely.

**Review by gavinbarron — APPROVED** [review]
Thanks for adding the examples @techienickb 

This looks good from my perspective

**Pull request merged**